strimzi/zookeeper-mode/clients-kustomize/overlays/http_internal/producer.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: whatever
+spec:
+  template:
+    spec:
+      containers:
+        - name: producer
+          env:
+            - name: MESSAGE
+              value: |
+                {
+                  "records": [
+                    {
+                      "key": "1234",
+                      "value": {
+                        "order_id": "1234",
+                        "amount": 99.99
+                      }
+                    }
+                  ]
+                }
+            - name: TOPIC
+              value: http-internal-data
+            - name: STRIMZI_SEND_INTERVAL
+              value: "1"  # 1000 msg/s
+            - name: STRIMZI_HOSTNAME
+              value: rivendell-bridge-bridge-service
```

Comment:
could you specify service with ns, etc ?

Reply:
done